src/functypedefrewriter.ts

```diff
@@ -4,6 +4,19 @@ module TsdPlugin {
      * A helper class to rewrite @typedef based function annotations into the @callback form
      */
     export class FunctionTypedefRewriter {
+        public static getDocletKind(doclet: jsdoc.IDoclet): string {
+            //Not a function or constructor eh?
+            if (doclet.kind != DocletKind.Function && doclet.kind != DocletKind.Class) {
+                if (doclet.params && doclet.params.length > 0) {
+                    doclet.memberof = null;
```

Comment:
Thanks for the find. I'll check if removing that line doesn't negatively impact OpenLayers